### ISSUE TITLE
Can't install directus with npx create-directus-project

### ISSUE THREAD

**Issue by vanch3d** [directus/directus#6848]
### Describe the Bug

Creation of a new Directus project fails:

```
$ npx create-directus-project directus-almotech-cms
Need to install the following packages:
  create-directus-project
Ok to proceed? (y)
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'path/posix'
Require stack:
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\@directus\shared\dist\cjs\utils\generate-extensions-entry.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\@directus\shared\dist\cjs\utils\index.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\extensions.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\services\graphql.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\services\index.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\utils\get-schema.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\cli\commands\bootstrap\index.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\dist\cli\index.js
- D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\directus\cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (D:\Dev\WebstormProjects\directus-almotech-cms\node_modules\@directus\shared\dist\cjs\utils\generate-extensions-entry.js:7:33)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Module.require (internal/modules/cjs/loader.js:974:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\@directus\\shared\\dist\\cjs\\utils\\generate-extensions-entry.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\@directus\\shared\\dist\\cjs\\utils\\index.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\extensions.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\services\\graphql.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\services\\index.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\utils\\get-schema.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\cli\\commands\\bootstrap\\index.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\dist\\cli\\index.js',
    'D:\\Dev\\WebstormProjects\\directus-almotech-cms\\node_modules\\directus\\cli.js'
  ]
}
× Installing Directus
Error: undefined

```

### To Reproduce

```npx create-directus-project directus-example ```

Tried on two settings:
- Windows 10 / Node v14.17.3
- Debian GNU/Linux 10 (buster) /  Node v13.14.0

Both gives me the same error

### What version of Directus are you using?

latest

### What version of Node.js are you using?

14.17.3 / 13.14.0

### What database are you using?

not reaching DB configuration

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10 / Debian 10

### How are you deploying Directus?

locally, command line

**Comment by rijkvanzanten** [comment]
What version of directus are you trying to install? This was an issue in rc 84 that should've been resolved in 85

**Comment by vanch3d** [comment]
> What version of directus are you trying to install? This was an issue in rc 84 that should've been resolved in 85

I tried it last at about the time RC85 was released so it might have been 84. Let me try again now.

**Comment by vanch3d** [comment]
I can confirm RC85 resolves the problem. Thanks!
